pkg/ovs/daemonset.go

```diff
@@ -175,8 +156,7 @@ func DaemonSet(
 								"/bin/bash", "-c",
```

Comment:
we no longer need && and so we can call to ovn-controller binary directly here, omitting a shell spawn.